deployment/network-operator/charts/sriov-network-operator/values.yaml

```diff
@@ -42,8 +42,8 @@ resources: {}
 
 # Image URIs for sriov-network-operator components
 images:
-  operator: quay.io/openshift/origin-sriov-network-operator:4.8
-  sriovConfigDaemon: quay.io/openshift/origin-sriov-network-config-daemon:4.8
+  operator: harbor.mellanox.com/cloud-orchestration/sriov-operator:latest
```

Comment:
harbor.mellanox.com is available only inside Mellanox network. Can we use public registry here?

Reply:
You have a point
@moshe010 should I change it to mellanox dockerhub

Reply:
yes let change to dockerhub

Reply:
also can you tag the images and not use latest

Reply:
updated